### ISSUE TITLE
base.cfg missing entries

### ISSUE THREAD

**Issue by brac3r** [DiscoveryGC/FLHook#196]
I was going to [implement this feature](https://github.com/DiscoveryGC/FLHook/issues/152) as it's something I also want and it will help you guys out as well, we all win.

Found LoadSettingsActual() in Main.cpp and figured out how I would do it. But I noticed a whole stack of variables are trying to get values that don't exist in the base.cfg file.

- damage_threshold
- siege_mode_damage_trigger_level
- no_crew_damage_multiplier
- shield_reinforcement_increment
- ...there are more, you get the idea

There's also a bunch missing in base_archtypes.cfg.
In short, the config files are not only out of date, which has enough problems to start with, but are also missing entries.

I'd like to politely ask again for the proper config files.
